pixi-multistyle-text.ts

```diff
@@ -75,6 +77,39 @@ export default class MultiStyleText extends PIXI.Text {
 		this.dirty = true;
 	}
 
+	public setTagStyle(tag: string, style: ExtendedTextStyle): void {
+		if (tag === "default") {
+			this.textStyles[tag] = this.assign(MultiStyleText.DEFAULT_TAG_STYLE, style);
+		} else {
+			this.textStyles[tag] = this.assign({}, style);
+		}
+
+		this._style = new PIXI.TextStyle(this.textStyles["default"]);
+		this.dirty = true;
+	}
+
+	public updateTagStyle(tag: string, style: ExtendedTextStyle): void {
+		if (tag in this.textStyles) {
+			this.assign(this.textStyles[tag], style);
```

Comment:
Hmm... Not sure I follow the reasoning behind both `set` and `update`. I'd rename this one as `set` and remove the old one. And I'd replace the content for something like this
```js
if ({}.hasOwnProperty.call(this.textStyles, tag)) {
  this.assign(this.textStyles[tag], MultiStyleText.DEFAULT_TAG_STYLE, this.textStyles[tag], style);
}
```